src/scss/cdb-components/forms/checkbox.scss

```diff
@@ -15,68 +27,83 @@
 @import '../../cdb-variables/colors';
 
 .CDB-Checkbox {
-  position: relative;
   width: $baseSize * 2;
   height: $baseSize * 2;
-  border: 1px solid $cMainLine;
-  border-radius: 3px;
-
-  &::before,
-  &::after {
-    display: block;
-    position: absolute;
-    bottom: 3px;
-    width: 2px;
-    border-radius: 1px;
-    background: $cSecondaryLine;
-    content: '';
-  }
+  opacity: 0;
+  position: absolute;
+  z-index: 1;
 
-  &::before {
-    right: $baseSize / 2;
-    height: $baseSize;
-    transform: rotate(45deg);
-  }
+  ~ .CDB-CheckboxFace {
```

Comment:
I came up to this resolution because I didn't find another way to link the states of the input (hover, disabled, checked) with the `.CDB-CheckboxFace` element, which makes the visual interaction. Should I use another operator or redefine the selectors?